main.go

```diff
@@ -120,7 +122,25 @@ func tagFileResources(path string, dir string, filter string, tags string, tfVer
 			} else {
 				log.Print("[INFO] Resource not taggable, skipping.", resource.Labels())
 			}
+		case "locals":
+			// Checks if terratag_added_* exists.
+			// If it exists no need to append it again to Terratag file.
+			// Instead should override it.
+			attributes := resource.Body().Attributes()
+			key := tag_keys.GetTerratagAddedKey(filename)
+			for attributeKey, attribute := range attributes {
+				if attributeKey == key {
+					mergedLocals, err := convert.MergeLocals(attribute, terratag.Added)
```

Comment:
This idea is to get the existing locals, merge it with terratag.Added. 
Remove the old locals, and append the new (merged) one.